swupd/packs.go

```diff
@@ -435,20 +495,38 @@ func CreatePack(name string, fromVersion, toVersion uint32, outputDir, chrootDir
 		return nil, err
 	}
 	var fromM *Manifest
+	var deltas []Delta
 	if fromVersion > 0 {
 		fromDir := filepath.Join(outputDir, fmt.Sprint(fromVersion))
 		fromM, err = ParseManifestFile(filepath.Join(fromDir, "Manifest."+name))
 		if err != nil {
 			return nil, err
 		}
+
+		// TODO: Make CreatePack itself take a Config.
+		var c config
+		c, err = getConfig(filepath.Join(outputDir, ".."))
+		if err != nil {
+			return nil, err
+		}
+
+		deltas, err = createDeltasFromManifests(&c, fromM, toM)
+		if err != nil {
+			return nil, err
+		}
+
+		// TODO: remove this.
+		for _, d := range deltas {
```

Comment:
I assume this *will* be removed for the final version of this PR?

Reply:
Sure. This PR was more in a WIP kind of stage. I'll remove this when I update it.